slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -0,0 +1,193 @@
+import asyncio
+import logging
+from asyncio import Future
+from asyncio import Queue
+from logging import Logger
+from typing import Union, Optional, List, Callable, Awaitable
+
+import aiohttp
+from aiohttp import ClientWebSocketResponse, WSMessage, WSMsgType, ClientConnectionError
+
+from slack_sdk.socket_mode.async_client import AsyncBaseSocketModeClient
+from slack_sdk.socket_mode.async_listeners import (
+    AsyncWebSocketMessageListener,
+    AsyncSocketModeRequestListener,
+)
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.web.async_client import AsyncWebClient
+
+
+class SocketModeClient(AsyncBaseSocketModeClient):
+    logger: Logger
+    web_client: AsyncWebClient
+    app_token: str
+    wss_uri: Optional[str]
+    auto_reconnect_enabled: bool
+    web_socket_message_queue: Queue
```

Comment:
I intentionally use `web_socket_` and `socket_mode_` prefix for field / method names to make the code clear but most of them look very long. We may want to remove `web_socket_` as much as possible as it's obvious.